src/js/pages/system/UnitsHealthDetail.js

```diff
@@ -47,6 +55,24 @@ class UnitsHealthDetail extends mixin(StoreMixin) {
     UnitHealthStore.fetchUnitNodes(this.props.params.unitID);
   }
 
+  onUnitHealthStoreUnitSuccess() {
+    let isLoading = this.state.isLoading & ~FLAGS.UNIT_LOADED;
```

Comment:
I don't see any need for bitwise operators here... You're dealing only with two flags, and I don't see any issue with a slightly bloated but easy digestible condition instead of a fancy obscure bitwise operation. Let's please not introduce them, as they are hard to understand (at least for most of the people) and inconsistent with the rest of our codebase.
How about a solution using a map or something similar that is, as well able to handle lots of flags without bloating the condition? 

```javascript
/**
* Check if all resources are loaded
* @param {Map} resourceMap
* @return boolean
*/
function areAllResourcesLoaded(resourceMap){
  return Array.from(resourceMap.values()).every(function ({loaded}}) {
  
    return loaded;
  });
}

// Init resource map
let resourceMap = new Map();
resourceMap.set('units', {loaded:false});
resourceMap.set('nodes', {loaded:false});
resourceMap.set('foo', {loaded:false})
resourceMap.set('bar', {loaded:false})

// Update map in load handler
resourceMap.set('units', {loaded:true});
resourceMap.set('nodes', {loaded:true});
resourceMap.set('foo', {loaded:true})
resourceMap.set('bar', {loaded:true})

// Check if everything is loaded
if(!areAllResourcesLoaded(resourceMap)){
    console.log('🕑  still loading')
}else {
   console.log('🎉  everything is loaded')
}
```
_The above is untested and should only illustrate a solution using a map or array_